packages/rollup/test/code_splitting/spec.js

```diff
@@ -6,6 +6,6 @@ describe('rollup code splitting', () => {
     const chunks = runfiles.resolvePackageRelative('bundle');
 
     expect(fs.existsSync(chunks + '/bundle.js')).toBeTruthy();
-    expect(fs.existsSync(chunks + '/chunk.js')).toBeTruthy();
```

Comment:
Seems like rollup no longer renames to "chunk", I assume because `strings.js` doesn't import anything else. Might be worth adding a more complicated code-splitting test...

Reply:
yeah this is the simple "unit test" level assertion, I think your change here is fine since we still check that it outputs two files rather than one